src/App.vue

```diff
@@ -50,50 +51,24 @@ export default {
       return this.mergedRoute.page
     },
     messages() {
-      return this.$store.getters['traction/messages']
+      // Access the messages from the root store
+      const rootStore = useRootStore()
```

Comment:
Can we set the store in `data() {}` to prevent reinitialising in each method (e.g. dismiss, clearMessaages)?

Reply:
I have moved the component to composition API. This does also remove what seem like some unused properties and methods.